graylog2-web-interface/src/components/lookup-tables/adapters/DnsAdapterFieldSet.tsx

```diff
@@ -132,8 +132,6 @@ class DnsAdapterFieldSet extends React.Component<
           unit={config.cache_ttl_override_unit || 'MINUTES'}
           units={['MILLISECONDS', 'SECONDS', 'MINUTES', 'HOURS', 'DAYS']}
           enabled={config.cache_ttl_override_enabled}
-          labelClassName="col-sm-3"
-          wrapperClassName="col-sm-9"
```

Comment:
If I remember correctly, we no longer style components this way so I removed them from the component as well.